content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md

```diff
@@ -14,67 +14,134 @@ import InfoBox from 'shared/components/InfoBox';
 
 
-# Prerequisites
+- VMs that use Persistent Volumes must have shared ``ReadWriteMany`` (``RWX``) access mode. For more information, refer to the [Persistent Volume Access Modes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) Kubernetes resource. VMs that do not use persistent storage, such as containerDisks, do not require modifications for live migration.
+
+
+- A VM’s pod network cannot use a Bridge interface. Disable the default Bridge interface on the pod network. However, other interfaces such as those that Multus grants, may use a bridge interface for live migration.
+
+
+# Migrate VM to a Different Node
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
 
-- VMs that use a Persistent Volume Claim (PVC) must have a shared ReadWriteMany (RWX) access mode. 
 
+2. From the left **Main Menu**, choose **Clusters** and click on your cluster. 
 
-- A VM’s pod network cannot use a bridge interface. Disable the bridge interface on the pod network.
 
+3. Click the **Virtual Machines** tab.
 
-<InfoBox>
 
-The default network interface type is a Bridge interface.
+4. Select the VM to migrate and use either the **three-dot Menu** or the **Actions drop-down Menu**, and click **Migrate Node to Node**.  
 
-</InfoBox>
+<br />
 
 
-# Enablement
+## Validation
 
 1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
 
 
-2. From the left **Main Menu**, click **Clusters** and click on your cluster. 
+2. From the left **Main Menu**, click **Clusters** and select the cluster. 
 
 
-3. Click the **Virtual Machines** tab.
+3. Navigate to  the **Virtual Machines** tab, and click the VM you migrated. 
 
 
-4. From the **three-dotMenu** or the **Actions drop-down Menu** for a selected VM, click **Migrate Node to Node**.
+4. Click the **Details** tab, and verify that the name and IP address of the new node is changed.
 
 
-# Validation
+# Maintain a Node
 
-1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+Compute nodes can be placed into maintenance mode using the `cordon` and `drain` commands. These commands mark the node as un-schedulable and drains all the virtual machines and pods from it. This process is useful in case you need to perform hardware maintenance on the node - for example to replace a disk or network interface card (NIC) card, perform memory maintenance, or if there are any issues with a particular node that need to be resolved. To learn more, check out the [Safely Drain a Node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#use-kubectl-drain-to-remove-a-node-from-service) Kubernetes resource.  
+
+
+## Prerequisites
+
+- Ensure `LiveMigrate` is set as the eviction strategy for all affected VMs. This provides seamless migration when the host is put in maintenance mode.   
```

Comment:
```suggestion
- Ensure `LiveMigrate` is set as the eviction strategy for all affected VMs. When the host is put in maintenance mode, this feature allows for a smooth and uninterrupted migration process.   
```
An alternative vs saying seamless or something indicating this is easy. We want to avoid such language as it is subjective.